src/App.tsx

```diff
@@ -1,31 +1,60 @@
+/* eslint-disable react/no-unused-state */
 import React from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
+import { getTodos } from './api';
+import { Todo } from './react-app-env';
+
 interface State {
   selectedUserId: number;
+  todos: Todo[];
 }
 
 class App extends React.Component<{}, State> {
   state: State = {
-    selectedUserId: 0,
+    selectedUserId: 1,
+    todos: [],
+  };
+
+  componentDidMount() {
+    this.load();
+  }
+
+  onSelect = (userId: number) => {
+    this.setState({ selectedUserId: userId });
   };
 
+  clear = () => {
+    this.setState({ selectedUserId: 0 });
+  };
+
+  async load() {
+    this.setState({ todos: await getTodos() });
+  }
+
   render() {
     const { selectedUserId } = this.state;
 
     return (
       <div className="App">
         <div className="App__sidebar">
-          <TodoList />
+          <TodoList
+            todos={this.state.todos}
+            selectedUserId={this.state.selectedUserId}
```

Comment:
Make use of destructuring
```suggestion
            todos={todos}
            selectedUserId={selectedUserId}
```

Reply:
Done, thanks